docs/v2.4/reference/intercepts.md

```diff
@@ -112,9 +177,17 @@ Finally, run `telepresence leave <name of intercept>` to stop the intercept.
 
 ## Creating an intercept when a service has multiple ports
 
-If you are trying to intercept a service that has multiple ports, you need to tell telepresence which service port you are trying to intercept. To specify, you can either use the name of the service port or the port number itself. To see which options might be available to you and your service, use kubectl to describe your service or look in the object's YAML. For more information on multiple ports, see the [Kubernetes documentation](https://kubernetes.io/docs/concepts/services-networking/service/#multi-port-services).
+If you are trying to intercept a service that has multiple ports, you
+need to tell Telepresence which service port you are trying to
+intercept.  To specify, you can either use the name of the service
+port or the port number itself.  To see which options might be
+available to you and your service, use kubectl to describe your
+service or look in the object's YAML. For more information on multiple
+ports, see the [Kubernetes documentation][kube-multi-port-services].
 
-```
+[kube-multi-port-services]: https://kubernetes.io/docs/concepts/services-networking/service/#multi-port-services
```

Comment:
Is the purpose of this to prevent long lines (e.g. on like 186) or is there some other significance?

Reply:
Yeah, just to make it more readable in a text editor.